packer/bsd/netbsd_postgres.pkr.hcl

```diff
@@ -10,23 +10,29 @@ variable "prefix" {
   default = ""
 }
 
+locals {
+  name = "${var.prefix}pg-ci"
+  version = "9-2"
+}
+
```

Comment:
Wonder if, as part of a separate change, we should add a common variable file for -vanilla and -postgres, which then could define the versions (including image URL & hash) in one place .

Reply:
That makes sense, creating an issue.